fuzz_test.go

```diff
@@ -37,7 +37,7 @@ func TestFuzzProveVerifyNameSpace(t *testing.T) {
 		for _, ns := range sortedKeys {
 			leafDataList := nidDataMap[ns]
 			for _, d := range leafDataList {
-				err := tree.Push(d[:size], d[size:])
+				err := tree.Push(d[:size])
```

Comment:
does this test only need namespaces now?

Reply:
Nope, good catch!